src/main/java/org/jpeek/App.java

```diff
@@ -68,12 +68,12 @@
  * @checkstyle MethodLengthCheck (500 lines)
  * @checkstyle JavaNCSSCheck (500 lines)
  *
- * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xml).
+ * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xsl).
```

Comment:
@uurl I've seen this line in 5 different pull requests. Please, use separate branches.